EnglishFilter/sections/foreign.txt

```diff
@@ -785,6 +785,8 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/35180
 sohu.com##div[data-spm-content="promotion"]
 sohu.com##a[href^="https://m.sohu.com/promotion?"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/36199
+||*.gif$third-party,domain=m.7ktw.com
```

Comment:
Do the domains for the ad change? I did not see a valid .gif when investigating the site.

So far sp0.baidu.com seems to include a tracking gif which is blocked with given rule.

Reply:
> Do the domains for the ad change?

I didn't see that. Found 2 domains `https://img.prscnz.cn`and `https://img.xiwnu.cn`. My rule is correct or i should change it?

Reply:
Leave it that way I think - both domains show me forbidden when visiting them and domain itself is not responding.

Reply:
`||*` is unnecessary in this case